Documentation/manual-olm-install.md

```diff
@@ -0,0 +1,141 @@
+
+Currently we require that metering is installed into its own namespace which requires some setup.
+
+First, start by creating the `openshift-metering` namespace:
+
+```
+kubectl create ns openshift-metering
+```
+
+Next a `CatalogSourceConfig` needs to be added to the `openshift-marketplace` namespace.
+This results in a `CatalogSource` containing the `metering` OLM package being created in the `openshift-metering` namespace.
+
+Download the `metering-operators` [metering.catalogsourceconfig.yaml][metering-catalogsourceconfig] and install it into the `openshift-marketplace` namespace:
+
+```
+kubectl apply -n openshift-marketplace -f metering.catalogsourceconfig.yaml
+```
+
+After it is created, confirm a new `CatalogSource` is created in the `openshift-metering` namespace:
+
+```
+kubectl -n openshift-metering get catalogsources
+NAME                 NAME     TYPE   PUBLISHER   AGE
+metering-operators   Custom   grpc   Custom      7s
+```
+
+You should also see a pod with a name resembling `metering-operators-12345` in the `openshift-marketplace` namespace, this pod is the package registry pod OLM will use to get the `metering` package contents:
+
+```
+kubectl -n openshift-marketplace get pods
+NAME                                    READY   STATUS    RESTARTS   AGE
+certified-operators-77b9986c5f-b7ndr    1/1     Running   0          25m
+community-operators-7d57677987-88477    1/1     Running   0          25m
+marketplace-operator-77dd7759cc-4thqm   1/1     Running   0          25m
+metering-operators-784f4ccccc-ctdxd     1/1     Running   0          43s
+redhat-operators-569bcbb8c5-7v2kw       1/1     Running   0          25m
+```
+
+Next, you will create an `OperatorGroup` in your namespace that restricts the namespaces the operator will monitor to the `openshift-metering` namespace.
+
+Download the `metering-operators` [metering.operatorgroup.yaml][metering-operatorgroup] and install it into the `openshift-metering` namespace:
+
+```
+kubectl -n openshift-metering apply -f metering.operatorgroup.yaml
+```
+
+Lastly, create a `Subscription` to install the metering-operator.
+
+Download the [metering.subscription.yaml][metering-subscription] and install it into the `openshift-metering` namespace:
+
+
+```
+kubectl -n openshift-metering apply -f metering.subscription.yaml
+```
+
+Once the subscription is created, OLM will create all the required resources for the metering-operator to run.
+This step takes a bit longer than others, within a few minutes the metering-operator pod should be created.
+Verify the metering-operator has been created and is running:
+
+```
+kubectl -n openshift-metering get pods
+NAME                                READY   STATUS    RESTARTS   AGE
+metering-operator-c7545d555-h5m6x   2/2     Running   0          32s
+metering-operators-bvpf7            1/1     Running   0          80s
```

Comment:
@chancez when I try it locally, I don't see the `metering-operators-xxx` pod.  I only see the following

```
pruan@MacBook-Pro ~/workspace/testcases/OLM/tmp $ oc get pods                                                                                                                          
NAME                                 READY   STATUS    RESTARTS   AGE
metering-operator-7f85b548fd-dpmgh   2/2     Running   0          4m
```

Reply:
Yeah, that was supposed to be updated in a previous PR, Ill fix it here.